tuned_lens/__main__.py

```diff
@@ -26,6 +26,11 @@ def run():
         help="Name of dataset to use. Can either be a local .jsonl file or a name "
         "suitable to be passed to the HuggingFace load_dataset function.",
     )
+    parent_parser.add_argument(
+        "--cpu-offload",
+        action="store_true",
+        help="Use CPU offloading. Must be combined with --fsdp.",
```

Comment:
What does this actually do?

Reply:
CPU offload automatically moves some parameters to the CPU during a forward pass to save VRAM, it's needed for some very large models